.changesets/fix_bryn_test_propagation.md

```diff
@@ -0,0 +1,5 @@
+### Datadog sampling fix ([PR #5788](https://github.com/apollographql/router/pull/5788))
+
+This PR ensures that datadog context is set, this may help with undersampling issues that users are seeing. 
```

Comment:
```suggestion
### Fix Datadog sampling ([PR #5788](https://github.com/apollographql/router/pull/5788))

The router's Datadog exporter has been fixed so that traces are sampled as intended.

Previously, the Datadog exporter's context may not have been set correctly, causing traces to be undersampled.
```